conf/prod.routes

```diff
@@ -1,5 +1,10 @@
 # Add all the application routes to the app.routes file
-->         /                             app.Routes
+->         /v1                           app.Routes
+->         /vP1                      app_vP1.Routes
+
+->         /v1/sandbox               sandbox.Routes
+->         /vP1/sandbox          sandbox_vP1.Routes
```

Comment:
Same here for vP1. Should we change it to be
/1
/P1
/1/sandbox
/P1/sandbox
instead?

Reply:
Personally when I see say `/1/foo` I wouldn't necessarily think that the `/1` refers to version 1 whereas I would think that with `/v1/foo`. So I think it's good as it is but don't mind changing it either. Let me know which.

Reply:
I think those are the 3 options:
* /v1/sandbox & /vP1/sandbox
* /version1/sandbox and /versionP1/sandbox
* /1/sandbox and /P1/sandbox
I like any of the 2 bottom ones. "vP1" looks weird to me.
@edgarjimenez what do you think?

Reply:
I actually have `v1` and `vP1` and I agree with Edin in that `/1/foo` is not as intuitive as `/v1/foo`. Furthermore, I think `version1/sandbox` is too verbose.
IMHO `/v1/sandbox` & `/vP1/sandbox` is a good compromise.

Reply:
ok if you think it is better let s go for that